app/services/decline_qts.rb

```diff
@@ -14,6 +14,16 @@ def call
     ActiveRecord::Base.transaction do
       application_form.update!(declined_at: Time.zone.now)
       ApplicationFormStatusUpdater.call(application_form:, user:)
+
+      creator = user.is_a?(String) ? nil : user
+      creator_name = user.is_a?(String) ? user : ""
+
+      TimelineEvent.create!(
+        event_type: "application_declined",
+        creator:,
+        creator_name:,
+        application_form:,
+      )
```

Comment:
Could we put these lines in a private method on the class? I think this `def call` method is getting quite long now.